### ISSUE TITLE
nvda does not read intelliSense results in vs2019

### ISSUE THREAD

**Issue by kianoosh-shakeri** [nvaccess/nvda#9896]
### Steps to reproduce:
open visual studio, for instance make a C# windows forms app project. After the creation is done, head to the text editor and try to add a button by code(without toolbox) somewhere that works for example in the class scope or constructor scope. Type but. It is expected to intelliSense show "Button" as the first result. Well it does, but nvda does not read the result. Not even that. It doesn't read any of the results with up and down too which I thought it'd do and to my surprise it didn't.
### Actual behavior:
NVDA does not read the intelliSense results in any way I tried.
### Expected behavior:
Read the intelliSense results without any problems as the user types or presses up and down arrows.
#### NVDA installed/portable/running from source:
installed: NVDA 2019.2Beta2, Portable: NVDA py3_staging latest threshold as of july8 2019
#### NVDA version:
2019.2beta2, Py3_staging latest threshold as of july 8 2019
#### Windows version:
windows 10 1903
#### Name and version of other software in use when reproducing the issue:
visual studio 2019 and 2017
#### Other information about your system:
Lenovo laptop
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
yes. Previous versions used to work fine with inttelliSense, if we count out the problem that the user had to press up or down to get nvda to read intelliSense results after the line of code is spoken in some languages such as python

**Comment by josephsl** [comment]
Hi, there is a promising pull request that might be able to cover this scenario. Thanks.

**Comment by DrSooom** [comment]
@josephsl: Are you talking about PR #9739?

**Comment by leonardder** [comment]
This looks similar to #8336

**Comment by francipvb** [comment]
Hello,

Can you elaborate what machine you're using?

I suspect that UIA problems are caused mainly by timeouts...

Thanks,

**Comment by leonardder** [comment]
@kianoosh-shakeri Can you still reproduce this with Visual Studio 2019 (16.4?)

**Comment by Adriani90** [comment]
Closing in favor of #8336 and the discussion goes further there. Here we don't have any updates from the initial author.
